projects/core/src/auth/user-auth/services/auth-multisite-isolation.service.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { inject, TestBed } from '@angular/core/testing';
+import { Observable, of } from 'rxjs';
+import { AuthMultisiteIsolationService } from './auth-multisite-isolation.service';
+import { BaseSiteService } from '../../../site-context/facade/base-site.service';
+import { BaseSite } from '../../../model/misc.model';
+
+const mockBaseSite: BaseSite = {
+  uid: 'test-basesite',
+  isolated: true,
+};
+
+const mockDecorator = `|${mockBaseSite?.uid}`;
+
+class MockBaseSiteService {
+  get(): Observable<BaseSite> {
+    return of(mockBaseSite);
+  }
+}
+
+describe('AuthMultisiteIsolationService', () => {
+  let service: AuthMultisiteIsolationService;
+  let baseSiteService: BaseSiteService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        AuthMultisiteIsolationService,
```

Comment:
nitpick: this service is providedIn: root. so no need to provide it explicitly here